### PR TITLE
5686 azurecore apireview

### PR DESCRIPTION
**Pull request by hemanttanwar**

### CODE REVIEW

File: sdk/core/azure-core/src/main/java/com/azure/core/http/HttpRequest.java [hunk 1]
```diff
@@ -130,6 +132,19 @@ public HttpRequest setHeader(String name, String value) {
         return body;
     }
 
+    /**
+     * Set the request content.
```
Comment (conniey) [inline sdk/core/azure-core/src/main/java/com/azure/core/http/HttpRequest.java]:
```suggestions
     * Sets the content of the HTTP request.
```

File: sdk/core/azure-core/src/main/java/com/azure/core/http/HttpRequest.java [hunk 2]
```diff
@@ -130,6 +132,19 @@ public HttpRequest setHeader(String name, String value) {
...
+     *
+     * @param content the request content
+     * @param charset @{link {@link Charset}} of the given {@code content}
```
Comment (conniey) [inline sdk/core/azure-core/src/main/java/com/azure/core/http/HttpRequest.java]:
```suggestion
     * @param charset The {@link Charset} of the given {@code content}. 
```

File: sdk/core/azure-core/src/main/java/com/azure/core/http/HttpRequest.java [hunk 3]
```diff
@@ -130,6 +132,19 @@ public HttpRequest setHeader(String name, String value) {
         return body;
     }
 
+    /**
+     * Set the request content.
+     *
+     * @param content the request content
```
Comment (conniey) [inline sdk/core/azure-core/src/main/java/com/azure/core/http/HttpRequest.java]:
```suggestion
     * @param content The HTTP request content.
```

File: sdk/core/azure-core/src/main/java/com/azure/core/http/HttpRequest.java [hunk 4]
```diff
@@ -130,6 +132,19 @@ public HttpRequest setHeader(String name, String value) {
...
+     * @param charset @{link {@link Charset}} of the given {@code content}
+     * @return this HttpRequest
```
Comment (conniey) [inline sdk/core/azure-core/src/main/java/com/azure/core/http/HttpRequest.java]:
```suggestion
     * @return The updated {@link HttpRequest}.
```

File: sdk/core/azure-core/src/main/java/com/azure/core/util/polling/PollResponse.java [hunk 5]
```diff
@@ -67,6 +67,22 @@
         public static OperationStatus fromString(String name) {
             return fromString(name, OperationStatus.class);
         }
+
+        /**
+         * An operation will be considered complete if it is in one of the following state:
+         * <ul>
+         *     <li>SUCCESSFULLY_COMPLETED</li>
```
Comment (conniey) [inline sdk/core/azure-core/src/main/java/com/azure/core/util/polling/PollResponse.java]: Use `@link` to link these statuses.

File: sdk/core/azure-core/src/main/java/com/azure/core/util/polling/PollResponse.java [hunk 6]
```diff
@@ -71,9 +71,9 @@ public static OperationStatus fromString(String name) {
         /**
          * An operation will be considered complete if it is in one of the following state:
          * <ul>
-         *     <li>SUCCESSFULLY_COMPLETED</li>
-         *     <li>USER_CANCELLED</li>
-         *     <li>FAILED</li>
+         *     <li>{@code SUCCESSFULLY_COMPLETED}SUCCESSFULLY_COMPLETED</li>
```
Comment (conniey) [inline sdk/core/azure-core/src/main/java/com/azure/core/util/polling/PollResponse.java]:
```java
{@link OperationStatus#SUCCESSFULLY_COMPLETED SUCCESSFULLY_COMPLETED}
```

Is what I meant. This links to the class.

**Comment by weshaggard** [comment]
I assume this is going to wait for merging until after we finish the current releases as it will break some inflight releases.

**Comment by JonathanGiles** [comment]
@hemanttanwar what is the status of this PR?

**Comment by hemanttanwar** [comment]
Closing this PR as it was done in another PR by Sameeksha .

**Pull request closed without merging**